third-party/linshare/src/test/java/org/apache/james/linshare/LinshareExtension.java

```diff
@@ -114,15 +115,17 @@ public Linshare getLinshare() {
     }
 
     public LinshareAPI getAPIFor(LinshareFixture.Credential credential) throws Exception {
-        return LinshareAPI.from(configurationWithJwtFor(credential));
+        return LinshareAPI.from(configurationWithBasicAuthnFor(credential));
     }
 
-    public LinshareConfiguration configurationWithJwtFor(LinshareFixture.Credential credential) throws Exception {
-        AuthorizationToken token = LinshareAPIForTesting.from(credential, linshare).jwt();
+    public LinshareAPI getAPIForDelegationAccount() throws Exception {
+        return LinshareAPI.from(configurationWithBasicAuthnFor(new LinshareFixture.Credential(technicalAccountUUID, TECHNICAL_ACCOUNT.getPassword())));
+    }
 
+    public LinshareConfiguration configurationWithBasicAuthnFor(LinshareFixture.Credential credential) throws Exception {
```

Comment:
s/configurationWithBasicAuthnFor/configurationWithBasicAuthFor ?

Reply:
For **delegaion api**. User can use **jwt()** but **delegation** is not only **basicAuth** for now

Reply:
Sorry that was not clear... I meant there is a typo mistake :) Authn -> Auth

Reply:
For you information, authn is the shortcut for authentication while authz is the one for authorization.